gcp/configuration/certificate.tf

```
@@ -0,0 +1,8 @@
+locals {
+  subdomain_name = format("%s.firetigerapi.com", local.deployment_name)
```

Should we use the bucket name instead?
```suggestion
  subdomain_name = format("%s.firetigerapi.com", var.bucket)
```